pkg/util/helper/workstatus.go

```diff
@@ -283,7 +283,7 @@ func IsWorkContains(workStatus *workv1alpha1.WorkStatus, targetResource schema.G
 }
 
 // BuildStatusRawExtension builds raw JSON by a status map.
-func BuildStatusRawExtension(status map[string]interface{}) (*runtime.RawExtension, error) {
+func BuildStatusRawExtension(status interface{}) (*runtime.RawExtension, error) {
```

Comment:
Just a question, why change the parameter here?

Reply:
So we can directly use this function when the input parameter is `appsv1.DeploymentStatus` or something else.

In fact, we will marshal the status inside this method, the input parameter of the `json.Marshal` is not restrict with `map[string]interface{}` type.

This change can makes this function more general.